website/addons/dataverse/static/dataverseUserConfig.js

```diff
@@ -123,6 +159,26 @@ function ViewModel(url) {
             }, timeout);
         }
     };
+
+    // Update observables with data from the server
+    $.ajax({
```

Comment:
Done.

I left in DataverseUserConfig, but I'm not sure what the backwards compatibility is referring to. Are there other places that interface with the view model?